yarn-project/simulator/src/avm/journal/journal.ts

```diff
@@ -253,11 +253,12 @@ export class AvmPersistableStateManager {
     if (success) {
       this.acceptNestedCallState(nestedState);
     }
-    const functionName =
+    let functionName =
       (await nestedState.hostStorage.contractsDb.getDebugFunctionName(
         nestedEnvironment.address,
         nestedEnvironment.temporaryFunctionSelector,
       )) ?? `${nestedEnvironment.address}:${nestedEnvironment.temporaryFunctionSelector}`;
+    createDebugLogger('aztec:simulator:external_calls').verbose(`[AVM] Calling nested function ${functionName}`);
```

Comment:
You should just be able to say `this.log.verbose`

Reply:
Addressed.